src/renderer/components/dock/create-resource.store.ts

```diff
@@ -1,13 +1,62 @@
+import fs from "fs-extra";
+import path from "path";
+import os from "os";
+import groupBy from "lodash/groupBy";
+import filehound from "filehound";
+import { watch } from "chokidar";
 import { autobind } from "../../utils";
 import { DockTabStore } from "./dock-tab.store";
 import { dockStore, IDockTab, TabKind } from "./dock.store";
 
 @autobind()
 export class CreateResourceStore extends DockTabStore<string> {
+
   constructor() {
     super({
       storageKey: "create_resource"
     });
+    fs.ensureDirSync(this.userTemplatesFolder);
```

Comment:
I don't think we should create this if it isn't created.

Reply:
https://github.com/lensapp/lens/pull/2327#discussion_r594432805

Reply:
Yes I know I said that. I guess I wasn't being clear. We can fix that later.

I meant to say that you could do something like the following:

```js
try {
    return await fs.promises.readDir(this.userTemplatesFolder, { withFileTypes: true });
} catch (error) {
    if (error.code === "ENOENT") { // dir doesn't exists
        return [];
    }

    throw error; // else propogate.
}
```